.github/workflows/test.yml

```
@@ -0,0 +1,68 @@
+name: Create test builds
+
+on:
+  # Uncomment this if build on every push is needed
+  # push:
+  #   branches-ignore: [main, beta, alpha] # skip for deployment branches.
+  #   tags-ignore: ['*']
+  pull_request:
```

```suggestion
on: [pull_request]
```

Simplified syntax is available